rules/windows/file_event/file_event_win_uac_bypass_wow64log.yml

```diff
@@ -0,0 +1,24 @@
+title: UAC Bypass Using IDiagnostic Profile - File
+id: 48ea844d-19b1-4642-944e-fe39c2cc1fec
+status: experimental
+description: Detects the creation of a file named "wow64log.dll" by "dllhost.exe" part of "IDiagnosticProfileUAC" UAC bypass technique
+author: Nasreddine Bencherchali
+references:
+    - https://github.com/Wh04m1001/IDiagnosticProfileUAC
+date: 2022/07/03
+logsource:
+    product: windows
+    category: file_event
+detection:
+    selection:
+        Image|endswith: '\DllHost.exe'
+        TargetFilename: 'C:\Windows\System32\wow64log.dll'
```

Comment:
wow64log.dll seen to be Redline.Stealer specific
```yaml
TargetFilename|startwith: 'C:\Windows\System32\'
TargetFilename|endswith: '.dll'
```

Reply:
Yes, when I read the references this name can very easily be change.

Reply:
Done